docs/components/file-upload.md

```diff
@@ -268,24 +161,22 @@ All of the above 3 events will emit a `JavaScript Array` of type `FileList`. Thi
 - `type` (returns the MIME type of the file)
 - `webkitRelativePath` (returns the path the URL of the File is relative to)
 
-<KCard>
-  <template v-slot:body>
-    <KFileUpload label="Upload File" :label-attributes="{ help: `Accepted file types: ${acceptedFileType.join(', ')}` }" :accept="acceptedFileType" @file-added="file => printData(file)" @file-removed="() => fileData = ''" />
-  </template>
-</KCard>
+
+<KFileUpload label="Upload file" :label-attributes="{ info: `Accepted file types: ${acceptedFileType.join(', ')}` }" :accept="acceptedFileType" @file-added="file => printData(file)" @file-removed="() => fileData = ''" />
 
 ```html
 <KFileUpload
-  label="Upload File"
+  label="Upload file"
   @file-added="file => printData(file)"
   @file-removed="() => fileData = ''"
 />
 ```
 
 <br/>
 <div>Emitted value:
-  <pre v-if="fileData.length" class="emitted-value">{{ `File Data:` }}
-    <div v-for="(file) in fileData">
+  <pre v-if="fileData.length" class="emitted-value">
```

Comment:
suggestion: can you add a background color for this `pre` element just on this docs page so we can tell it's displaying something?